src/cljs/snapshot/lumo/repl.cljs

```diff
@@ -811,6 +811,19 @@
   (deps/index-upstream-foreign-libs))
 
 ;; --------------------
+;; Introspection
+
+(defn ^:export get-arglists
+  "Return the argument lists for the given symbol as string."
+  [s]
+  (when-let [var (some->> s repl-read-string first (resolve-var @env/*compiler*))]
+    (if-not (:macro var)
+      (:arglists var)
+      ;; filtering &env &form for macros
+      (map #(filterv (complement '#{&env &form}) %)
```

Comment:
For reference on this, I opened [CLJS-1958](http://dev.clojure.org/jira/browse/CLJS-1958), but it is probably better to have it here, not upstream I guess (see comment).